src/libraries/System.Runtime/tests/System.Runtime.InteropServices.RuntimeInformation.Tests/DescriptionNameTests.cs

```diff
@@ -196,17 +196,10 @@ public void VerifyFrameworkDescriptionContainsCorrectVersion()
                 return;
 
             Assert.DoesNotContain("+", version); // no git hash
+            Assert.DoesNotContain("-", version); // no preview suffix
```

Comment:
It was possible to identify preview or private builds from the FrameworkDescription string before this change. This feature has been useful for diagnostics. Should this feature be preserved?

Reply:
@jkotas yeah, we seem to have other tests that make that assumption so it was easier to restore the earlier behavior.